optimum/amd/ryzenai/modeling.py

```diff
@@ -181,11 +176,11 @@ def load_model(
         else:
             providers_options = None
 
-        is_dynamic = RyzenAIModel._check_uses_static_shape(path)
-        if is_dynamic and provider == "VitisAIExecutionProvider":
-            raise ValueError(
-                "The model provided has dynamic axes in input/output. Please provide model with static shapes for inference with RyzenAI."
-            )
+        # is_dynamic = RyzenAIModel._check_uses_static_shape(path)
+        # if is_dynamic and provider == "VitisAIExecutionProvider":
+        #     raise ValueError(
+        #         "The model provided has dynamic axes in input/output. Please provide model with static shapes for inference with RyzenAI."
+        #     )
```

Comment:
Should this be removed? Why is it commented out?

Reply:
In  their documentation, its mentioned
```
For CNN’s on NPU platform, dynamic input shapes are currently not supported and only a batch size of 1 is allowed. Please ensure that the shape of input is a fixed value, and the batch dimension is set to 1.
```

But since, LLMs support dynamic shapes had to comment it. Shifted the error to  image-classification model as we do not  support transformers model there yet.